resources/js/components/ExceptionsTable.vue

```diff
@@ -39,24 +39,25 @@
                     <div v-else>{{ __('Closed') }}</div>
                 </table-column>
                 <table-column v-if="editable" class="text-right">
-                    <default-button @click.prevent="$emit('addInterval', 'exceptions', exception.date)"><span class="px-1">+</span></default-button>
-                    &nbsp;
-                    <danger-button @click.prevent="$emit('removeException', exception.date)"><span class="px-1">-</span></danger-button>
+                    <add-button @click.prevent="$emit('addInterval', 'exceptions', exception.date)" />                    &nbsp;
```

Comment:
The `&nbsp;` is floating on the right there, can it be brought into the next line?